src/test/java/ezvcard/io/roundtrip/XCardRoundTripTest.java

```diff
@@ -0,0 +1,82 @@
+package ezvcard.io.roundtrip;
+
+import java.io.*;
+
+import org.junit.Test;
+
+import ezvcard.VCardVersion;
+import ezvcard.io.StreamReader;
+import ezvcard.io.StreamWriter;
+import ezvcard.io.xml.XCardReader;
+import ezvcard.io.xml.XCardWriter;
+
+public class XCardRoundTripTest extends RoundTripTestBase {
+
+	public XCardRoundTripTest() throws Exception {
+		updateSamples(VCardVersion.V4_0,
+				"outlook-2003" // &#12; in fburl is not valid xml
+		);
+		updateSamples(VCardVersion.V3_0);
+	}
+
+	@Test
+	public void equals_compare_vcard_4_to_xcard() throws Exception {
+		convertAllFromVCard(VCardVersion.V4_0, true, false,
+				"outlook", // newlines not preserved on linux
```

Comment:
I believe that the data itself is being changed. For instance, the outlook-2003.vcf contains "NOTE;ENCODING=QUOTED-PRINTABLE:This is the note field!!=0D=0ASecond line=0D=0A=0D=0AThird line is empty=0D=", which, if I understand correctly, the "=0D=0A" represent returns and newlines in the note text provided by the user, which are then converted to platform-default newlines, and written as "NOTE:This is the note field!!\nSecond line\n\nThird line is empty\n" in the converted vcf.